### PR TITLE
APP-4131 Focus at the top of a the dropdown list when the user press the 'home' key

### PR DESCRIPTION
**Pull request by symphony-annaLlorens**
### Description
 -  Focus at the top of a the dropdown list when the user press the 'home' key
 - Added UT

### Demo
May-31-2021 12-03-37

[Jira-ticket](https://perzoinc.atlassian.net/browse/APP-4131)

### CODE REVIEW

**Review by symphony-adnane — APPROVED** [review]

File: spec/components/dropdown/Dropdown.spec.tsx [hunk 1]
```diff
@@ -175,18 +176,39 @@ describe('Dropdown component test suite =>', () => {
...
+        const {  getByText } = render(
+          <Dropdown
+            options={dropdownProps.options}
+            displayArrowIndicator
+            enableTermSearch
+          />
+        );
+        const input = screen.getByRole('textbox');
+        userEvent.type(input,'B');
+        expect(getByText('banana').className).toContain('tk-select__option--is-focused');
```
Comment (AliciaSymphony) [inline spec/components/dropdown/Dropdown.spec.tsx:192]: to not rely on the class, you can also compare your element to document.active

**Review by AliciaSymphony — APPROVED** [review]
LGTM

File: spec/components/dropdown/Dropdown.spec.tsx (hunk 1 above)
Comment (symphony-annaLlorens) [reply spec/components/dropdown/Dropdown.spec.tsx:192]: The active element is the input from the dropdown.

Comment (AliciaSymphony) [reply spec/components/dropdown/Dropdown.spec.tsx:192]: ooooh! make sense

**Pull request merged**